samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/ble_mesh.c

```diff
@@ -10,7 +10,7 @@
 
 #ifdef OOB_AUTH_ENABLE
 
-static int output_number(bt_mesh_output_action_t action, uint32_t number)
+static int output_number(bt_mesh_output_action_t action, u32_t number)
```

Comment:
Wasn't this already fixed upstream? Could you try rebasing your PR on latest upstream master?

Reply:
I will keep things as it is.

Reply:
No, it wasn't get fixed in upstream

Reply:
I was looking at commit 4208642a536235477019b802f313b9802d16f2c9

But seems you're right: this commit missed the place you're fixing

Reply:
Done !!